docs/package.json

```diff
@@ -39,6 +39,7 @@
     "@material-ui/system": "5.0.0-alpha.24",
     "@material-ui/types": "5.1.6",
     "@material-ui/unstyled": "5.0.0-alpha.24",
+    "@material-ui/data-grid": "^4.0.0-alpha.18",
```

Comment:
I noticed that the other MUI packages are locked, should this one be locked as well?

Reply:
No, docs packages don't need to be locked.